.github/workflows/ga_delete-review-app.yml

```diff
@@ -0,0 +1,18 @@
+name: Delete Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
```

Comment:
```suggestion
```
We do not have a `main` branch, our trunk branch is called `master`. I think we would want to have a review app created against all PRs, not just those opened against `master`.